mmgen/models/architectures/stylegan/modules/styleganv2_modules.py

```diff
@@ -354,13 +356,18 @@ def forward(self, x, style):
         # process style code
         style = self.style_modulation(style).view(n, 1, c, 1,
                                                   1) + self.style_bias
-
         # combine weight and style
         weight = weight * style
         if self.demodulate:
             demod = torch.rsqrt(weight.pow(2).sum([2, 3, 4]) + self.eps)
             weight = weight * demod.view(n, self.out_channels, 1, 1, 1)
 
+        # TODO: input gain added by yyf
```

Comment:
We can remove this `TODO` if this modification has been verified.

Reply:
done